specs/binding-mqtt.spec/src/main/resources/META-INF/zilla/mqtt.idl

```diff
@@ -18,7 +18,8 @@ scope mqtt
     enum MqttSessionFlags (uint8)
     {
         CLEAN_START(1),
-        WILL(2)
+        WILL(2),
+        SHARDING(3)
```

Comment:
I think this would be better moved to `MqttServerCapabilities` and renamed.

```
        enum MqttServerCapabilities (uint8)
        {
            RETAIN (0),
            WILDCARD (1),
            SUBSCRIPTION_IDS (2),
            SHARED_SUBSCRIPTIONS (3),
            REDIRECT(4)
        }
```
where `REDIRECT` describes a capability of the `mqtt` server, meaning we can send the `RESET` with `MqttResetEx` saying to redirect to a different `serverRef`, and as it turns out we then happen to use that for `sharding` as an implementation detail at `mqtt-kafka`.